app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -227,6 +239,31 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
     fullNameTextView.text = resources.getString(R.string.patientsummary_name, name, genderLetter, age)
   }
 
+  private fun displayBpPassport(bpPassport: Optional<BusinessId>, isPhoneNumberVisible: Boolean) {
+    bpPassportTextView.visibility = if (bpPassport.isNotEmpty()) View.VISIBLE else View.GONE
```

Comment:
We already have a utility method `View.visibleOrGone(Boolean)` that can be used to do this sort of thing, FYI.